commitlint.config.js

```diff
@@ -1 +1,26 @@
-module.exports = { extends: ['@commitlint/config-conventional'] };
+// @see https://github.com/conventional-changelog/commitlint/blob/v12.0.1/@commitlint/config-conventional/index.js
+module.exports = {
+    extends: ['@commitlint/config-conventional'],
+
+    rules: {
+        'body-max-line-length': [1, 'always', 80],
+        'header-max-length': [1, 'always', 72],
+        'scope-case': [2, 'never', ['kebab-case', 'snake-case']],
+        'subject-case': [
+            2,
+            'never',
+            [
+                'upper-case',
+                'camel-case',
+                'kebab-case',
+                'pascal-case',
+                'snake-case',
+            ],
+        ],
+        'type-enum': [
+            2,
+            'always',
+            ['chore', 'docs', 'feat', 'fix', 'refactor', 'style', 'test'],
```

Comment:
@lkraav what do you think about adding `version` here as well. It's useful when we bump plugin or theme version by editing comment. 

On [upstream](https://github.com/conventional-changelog/commitlint/blob/v12.0.1/@commitlint/config-conventional/index.js#L23) closest to it is `build`

Reply:
I would avoid inventing types. Release sounds like `chore`, although.. have you looked around for how do others type their release commits with commitlint?

Reply:
> I would avoid inventing types. Release sounds like `chore`, although.. have you looked around for how do others type their release commits with commitlint?

`chore` should be enough for release commit.